packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -175,7 +175,7 @@ const props = defineProps({
     default: (({ items, query }) => {
       query = query.toLowerCase()
 
-      return items.filter((item) => item.tag && item.tag.toLowerCase().includes(query))
+      return items.filter((item) => item.tag && String(item.tag).toLowerCase().includes(query))
```

Comment:
```suggestion
      return items.filter((item) => item.tag && String(item.tag || '').toLowerCase().includes(query))
```

question: can `item.tag` ever be `undefined` or `null`, if so, recommend falling back to empty string.